middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
Sure, no problem.  Seems like we are in consensus about using net.SplitHostPort, didn't use it originally because I thought it was overkill, but I understand why you want to use it.  Are we in consensus about removing the checking of common headers used in proxy passing situations?  I will update this branch with these changes.